src/client/refactor/proxy.ts

```diff
@@ -39,8 +39,11 @@ export class RefactorProxy extends vscode.Disposable {
         // get line count
         // Rope always uses LF, instead of CRLF on windows, funny isn't it
         // So for each line, reduce one characer (for CR)
+        // But Not all Windows users use CRLF
         const offset = document.offsetAt(position);
-        return offset - position.line;
+
+        const winEols = getWindowsLineEndingCount(document.getText(), offset);        
+        return offset - winEols;
```

Comment:
i'm agree, i will update my Pull request later. Thank your for your suggestion. :smile: